### PR TITLE
Header & footer

### PR DESCRIPTION
**Pull request by Manpreet-hub**
- Added Header & Footer
- Added Sidebar section

Screenshot (115)

### CODE REVIEW

File: src/components/routes/Routers.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Route, Routes } from "react-router-dom";
+import { Home, Notes } from "../../pages/";
+import { Trash } from "../Trash/Trash";
+import { Archives } from "../Archives/Archives";
+import { Labels } from "../Labels/Labels";
+import { Note } from "../Notes/Note";
+
```
Comment (ajay-mandaviya) [inline src/components/routes/Routers.jsx:7]: you already import   this  component into your index.js file , so export this component into index.js file 
and  import this from index.js file

**Review by ajay-mandaviya — APPROVED** [review]
Everything looks good and well structure

**Review by abhijeetekad — COMMENTED** [review]
Code looks good to me, clean and well structured. good to merge

File: src/components/routes/Routers.jsx (hunk 1 above)
Comment (Manpreet-hub) [reply src/components/routes/Routers.jsx:7]: yes , will do it in the next PR.

**Pull request merged**